src/com/vauff/maunzdiscord/features/MapTimer.java

```diff
@@ -20,10 +20,18 @@
 import sx.blah.discord.handle.obj.IUser;
 import sx.blah.discord.util.EmbedBuilder;
 
+/**
+ * A timer to send notifications of maps currently being played on given servers
+ */
 public class MapTimer
 {
+	/** The servers to check */
```

Comment:
This HashMap is only used by other classes to get extended server info which doesn't get saved in serverInfo.json (e.g. *players command to get player list). It's never used in the timer to check any server info, instead calls are made to the server object directly.